src/NLog/Layouts/LayoutParser.cs

```diff
@@ -386,7 +386,10 @@ private static LayoutRenderer ParseLayoutRenderer(ConfigurationItemFactory confi
                         var value = ParseParameterValue(stringReader);
                         if (!string.IsNullOrEmpty(parameterName) || !StringHelpers.IsNullOrWhiteSpace(value))
                         {
-                            // TODO NLog 5.0 Should throw exception when invalid configuration (Check throwConfigExceptions)
+                            if (throwConfigExceptions ?? LogManager.ThrowConfigExceptions == true)
```

Comment:
Why not: `if (throwConfigExceptions ?? LogManager.ThrowConfigExceptions ?? LogManager.ThrowExceptions)` ?